Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ruby:2.2.3
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
I added it b/c execjs causes `Could not find a JavaScript runtime` errors without some runtime. Not exactly sure what is going on; the rails_admin/coffee-script or uglifier packages are require execjs.